Resources/Textures/Objects/Vehicles/atv.rsi/meta.json

```diff
@@ -5,7 +5,7 @@
     "y": 32
   },
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/40d89d11ea4a5cb81d61dc1018b46f4e7d32c62a",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/40d89d11ea4a5cb81d61dc1018b46f4e7d32c62a, resprited by decortes (discord)",
   "states": [
     {
       "name": "vehicle",
```

Comment:
You renamed the file to `Vehicle.png` from `vehicle.png`. Not sure, but I believe this doesn't work.

Reply:
Done

Reply:
github struggles with changes between vehicle.png and Vehicle.png Might need to push a change to a completely different word before pushing the lowercase version. Unless you managed to fix it.